src/main/java/com/learning/configuration/PropertiesReader.java

```diff
@@ -0,0 +1,35 @@
+package com.learning.configuration;
+
+import com.google.inject.Singleton;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Singleton
+public class PropertiesReader {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(PropertiesReader.class);
+  private final String pathToFile;
+
+  public PropertiesReader() {
+    pathToFile = "src/main/resources/application-properties.yaml";
+  }
+
+  public String getProperty(String property) {
+    String readProperty = "";
+    try (InputStream input = new FileInputStream(pathToFile)) {
```

Comment:
I suggest you get properties inputStream from classLoader instead, this will make it safer to use across different OS versions

Reply:
Why it is safer? What are possible issues? 
The best would be article so I can study it deeply :)

Reply:
Correct, I'm concerned about storing path this way and referencing it.
It's not a big deal, but in case of refactoring around this functionality might get overlooked, using `this.getClass().getClassLoader().getResourcesAsStream("");` assures path relative to resources, OS agnostic.